resurfemg/postprocessing/quality_assessment.py

```diff
@@ -193,3 +194,72 @@ def percentage_under_baseline(
     valid_timeproducts = percentages_aub < aub_threshold
 
     return valid_timeproducts, percentages_aub
+
+
+def consecutive_manoeuvres(
+        signal,
+        start_s,
+        end_s,
+        fs,
+        manoeuvres_idxs
+):
+    """
+    Detect manoeuvres (for example Pocc) with no supported breaths
+    in between. First identify the supported breaths from the ventilator signal
+    and then check for in-between supported breaths.
+    If no supported breaths are detected in between two manoeuvres,
+    a 'double dip' is detected.
+    Double_dips true means the manoeuvre is not valid.
+
+    :param signal: signal in which the manoeuvre is performed
+    :type signal: ~numpy.ndarray
+    :param start_s: start second of the window in which to be searched
```

Comment:
In previous function start_s is a sample number, which makes the function independent on fs. I would rephrase start_s (and end_s) to samples. 
You will still need fs for the peak width, but you could also take the peak_width (and height and prominence) as optional inputs